src/ResourceManager/ServiceFabric/AzureRM.ServiceFabric.psd1

```diff
@@ -75,13 +75,13 @@ FunctionsToExport = @()
 CmdletsToExport = 'Add-AzureRmServiceFabricApplicationCertificate', 
                'Add-AzureRmServiceFabricClientCertificate', 
                'Add-AzureRmServiceFabricClusterCertificate', 
-               'Add-AzureRmServiceFabricNode', 'Add-AzureRmServiceFabricNodeType', 
+               'Add-AzureRmServiceFabricNode', 
+               'Add-AzureRmServiceFabricNodeType', 
                'Get-AzureRmServiceFabricCluster', 
                'New-AzureRmServiceFabricCluster', 
                'Remove-AzureRmServiceFabricClientCertificate', 
                'Remove-AzureRmServiceFabricClusterCertificate', 
                'Remove-AzureRmServiceFabricNode', 
-               'Remove-AzureRmServiceFabricNodeType', 
```

Comment:
what's the recommend way to disable/remove the command , we have bug which can't be fix easily now, so we need more time to fix it , so we can enable this command again.